code/game/area/areas.dm

```diff
@@ -420,12 +421,26 @@ var/list/mob/living/forced_ambiance_list = new
 			continue
 		if (istype(A, /area/constructionsite))
 			continue
+		if (istype(A, /area/turbolift))
+			continue
 
 		//Although hostile mobs instadying to turrets is fun
 		//If there's no AI they'll just be hit with stunbeams all day and spam the attack logs.
 		if (istype(A, /area/turret_protected) || LAZYLEN(A.turret_controls))
 			continue
 
+		var/should_continue = FALSE

```

Comment:
`should_continue` has no meaning outside of the next sentence's scope. Ergo, it should be within the if-block starting at line 433.